testutils/channel_opts.go

```diff
@@ -193,6 +193,13 @@ func (o *ChannelOpts) SetRelayHosts(rh relay.Hosts) *ChannelOpts {
 	return o
 }
 
+// SetRelayLocal sets the channel's relay local handlers for service names
+// that should be handled by the relay channel itself.
+func (o *ChannelOpts) SetRelayLocal(relayLocal []string) *ChannelOpts {
+	o.ChannelOptions.RelayLocalHandlers = relayLocal
+	return o
+}
```

Comment:
If we're aiming for a fluent API here, would it be better to take a variadic number of strings and append them to any existing relay-local handlers?